recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+numpy:     # [aarch64]
+  - '1.16' # [aarch64]
+
+channel_targets:
+  - conda-forge rc
```

Comment:
All you need is to change the name here and rerender.
We also need to create an `rc` branch for it.

Reply:
Yeah, happy to submit this as a PR to a `rc` branch - just can't create it myself, apparently.

Reply:
Implemented as #25